geoportal/tests/test_raster.py

```diff
@@ -211,8 +211,8 @@ def test_profile_csv(self):
             '"coordinates":[[548009.5,215990],[547990,216009.5]]}'
         response = profile.csv()
         self.assertEqual(response.body.decode("utf-8"), """distance,dem,dem2,dem4,x,y
-0.0,1166,1166,-9999,548009.5,215990.0
-9.2,1175,1175,-9999,548003.0,215996.5
+0.0,1166,1166,None,548009.5,215990.0
+9.2,1175,1175,None,548003.0,215996.5
```

Comment:
This shouldn't change...

Reply:
I do think is has to change => https://github.com/camptocamp/c2cgeoportal/blob/master/geoportal/c2cgeoportal_geoportal/views/profile.py#L66
=> i will remove the code that sets -9999